x-pack/solutions/security/plugins/security_solution/public/attack_discovery/pages/use_find_attack_discoveries/index.ts

```diff
@@ -83,6 +85,7 @@ export const useFindAttackDiscoveries = ({
         method: 'GET',
         version: API_VERSIONS.internal.v1,
         query: {
+          alert_ids: alertIds,
```

Comment:
This PR is not tagged as being backported to `8.19`, but consider backporting this change to simplify future backports and testing, (avoiding diverging implementations).